### PR TITLE
Add new Starlette logo to Docs

### PR DESCRIPTION
**Pull request by koddr**
Add new Starlette logo to Docs

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <p align="center">
-  <img width="320" height="192" src="https://raw.githubusercontent.com/encode/starlette/master/docs/starlette.png" alt='starlette'>
+  <img width="100%" src="https://user-images.githubusercontent.com/11155743/55277958-02552b00-5317-11e9-8780-ef9874d137b2.jpg" alt='starlette'>
```
Comment (lovelydinosaur) [inline docs/index.md]: Rather than changing the location here, I'm happy for the logo *itsself* to replace `docs/starlette.png` for this pull request.

Comment (koddr) [reply docs/index.md]: Oh, I understand.

* I push new logo to `docs/starlette.png`;
* And change `` width to 100% in `./README.md` + `./docs/index.md`;

**Comment by lovelydinosaur** [comment]
Gonna try giving this a whirl. Great work!

**Comment by lovelydinosaur** [comment]
I've pulled down the width so as not to swamp the page. Looks great!
Might be nice to bump up the text/logo size relative to the star background? Is that something you'd be up for doing?

**Pull request merged**